python/source/data.rst

```diff
@@ -182,6 +182,54 @@ We can combine them into a single table using :func:`pyarrow.concat_tables`:
   cast data from one type to another (if `promote=True`).  In such cases the data 
   will need to be copied and an extra cost will occur.
 
+Adding a column to an existing Table
+====================================
+
+If you have a table it is possible to extend its columns using
+:meth:`pyarrow.Table.append_column`
+
+Suppose we have a table with oscar nominations for each actress
```

Comment:
```suggestion
Suppose we have a table with Oscars won by each actress
```

Reply:
Are you sure that's the case? I doubt a single person won 21 oscars. 21 seems to be the number of nominations for Meryl Streep.

Reply:
Oh, I misread and thought @amol- was using the same dataset as the one from the R cookbook, but you're right - this is a different one.